Week04/lesson08/homework/temp_converter/js/indexbonus3.js

```diff
@@ -0,0 +1,57 @@
+function conversion() {
+
+  var c = document.querySelector("#c").value;
+  var f = document.querySelector("#f").value;
+
+  if ((f === null || f === "") && (c === null || c === "")) {
+      document.querySelector("#alert").innerHTML = "Please fill one of the fields above.";
+  }
+
+  else if (f === null || f === "") {
+    f = c * 9 / 5 + 32;
+    document.querySelector("#f").value = Math.round(f*10)/10+"°F";
+    c = c+"°C";
+  }
+
+  else if (c === null || c === "") {
+    c = (f - 32) * 5 / 9;
+    document.querySelector("#c").value = Math.round(c*10)/10+"°C";
```

Comment:
That's a good point! Hmmmm... I'm sure there's some way so that whole number are rounded `32` but then long decimals are still taken into account but rounded to the second place (so `32.45555555` becomes `32.46`.. I'll look into it